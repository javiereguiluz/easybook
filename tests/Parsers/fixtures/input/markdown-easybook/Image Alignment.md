// regular image not aligned
![Test image](figure1.png)

// "alt text" has a whitespace on the left -> the image is left aligned
![ Test image](figure1.png)

// "alt text" has a whitespace on the right -> the image is right aligned
![Test image ](figure1.png)

// "alt text" has whitespaces both on the left and on the right -> the image is centered
![ Test image ](figure1.png)

If you enclose alt text with quotes, make sure that whitespaces are placed
outside the quotes. The following images for example don't define any alignment:

!["Test image"](figure1.png)

![" Test image"](figure1.png)

!["Test image "](figure1.png)

![" Test image "](figure1.png)

Image alignment is also possible when using the alternative image syntax:

![Test image][1]

![ Test image][1]

![Test image ][1]

![ Test image ][1]

[1]: figure1.png