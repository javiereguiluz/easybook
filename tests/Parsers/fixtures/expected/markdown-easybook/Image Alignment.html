<p>// regular image not aligned
<img src="figure1.png" alt="Test image" /></p>

<p>// "alt text" has a whitespace on the left -> the image is left aligned
<div class="left"><img src="figure1.png" alt="Test image" /></div></p>

<p>// "alt text" has a whitespace on the right -> the image is right aligned
<div class="right"><img src="figure1.png" alt="Test image" /></div></p>

<p>// "alt text" has whitespaces both on the left and on the right -> the image is centered
<div class="center"><img src="figure1.png" alt="Test image" /></div></p>

<p>If you enclose alt text with quotes, make sure that whitespaces are placed
outside the quotes. The following images for example don't define any alignment:</p>

<p><img src="figure1.png" alt="&quot;Test image&quot;" /></p>

<p><img src="figure1.png" alt="&quot; Test image&quot;" /></p>

<p><img src="figure1.png" alt="&quot;Test image &quot;" /></p>

<p><img src="figure1.png" alt="&quot; Test image &quot;" /></p>

<p>Image alignment is also possible when using the alternative image syntax:</p>

<p><img src="figure1.png" alt="Test image" /></p>

<p><div class="left"><img src="figure1.png" alt="Test image" /></div></p>

<p><div class="right"><img src="figure1.png" alt="Test image" /></div></p>

<p><div class="center"><img src="figure1.png" alt="Test image" /></div></p>
